Tools/CI/MissionCheck.py

```diff
@@ -0,0 +1,205 @@
+################################################################################################
+# Settings
+################################################################################################
+
+import_mission_filename = "VTOLmission.txt"
+max_execution_time      = 250
+
+
+################################################################################################
+# Init
+################################################################################################
+
+# Import DroneKit-Python
+from dronekit import connect, VehicleMode, Command
+import time, sys
+
+# start time counter
+start_time = time.time()
+elapsed_time = time.time() - start_time
+
+# Connect to the Vehicle, wait_ready will not work on PX4
+print "Connecting"
+vehicle = connect('127.0.0.1:14550', wait_ready=True)
```

Comment:
Does it still work if you use 14540 here? That would connect to the off board control port and leave the other one open to check the progress with QGroundControl.